staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -236,6 +241,8 @@ func DeleteCollection(r rest.CollectionDeleter, checkBody bool, scope *RequestSc
 					scope.err(err, w, req)
 					return
 				}
+				// For backwards compatibility, we need to allow existing clients to submit per group DeleteOptions
+				// It is also allowed to pass a body with meta.k8s.io/v1.DeleteOptions
```

Comment:
by the time we get here, I thought whatever group/version the user provided was consumed and the options object was just a &metav1.DeleteOptions{} struct with no typemeta. Is that not the case?

Reply:
Yeah, I just copy-pasted this over from Delete() for consistency.